src/main/java/jewellery/inventory/dto/request/UserRequestDto.java

```diff
@@ -29,4 +42,23 @@ public class UserRequestDto {
       regexp = "^(?=.*[0-9])(?=.*[a-z])(?=.*[A-Z])(?=.*[@#$%^&+=])(?=\\S+$).{8,}$",
       message = PWD_PATTERN_VALIDATION_MSG)
   private String password;
+
+  @Size(min = 3, max = 100, message = "Address length must be between 3 and 100")
+  private String address;
+
+  @Pattern(regexp = PHONE_REGEX, message = PHONE_PATTERN_VALIDATION_MSG)
+  private String phone;
+
+  @Pattern(regexp = PHONE_REGEX, message = PHONE_PATTERN_VALIDATION_MSG)
+  private String phone2;
+
+  @DateTimeFormat(pattern = "dd/MM/yyyy")
+  @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "dd/MM/yyyy")
+  @PastOrPresent(message = "Birth date must be a past or present date")
+  private LocalDate birthDate;
+
+  private String favouriteColor;
```

Comment:
Turns out we don't need favouriteColor as well

Reply:
Did not know that. Removed.